tests/templates/kuttl/mount-dags-gitsync/30-install-airflow-cluster.yaml.j2

```diff
@@ -34,9 +34,8 @@ spec:
 {% endif %}
     credentialsSecret: test-airflow-credentials
     dagsGitSync:
-      - name: git-sync
-        repo: https://github.com/stackabletech/airflow-operator
-        branch: "main"
+      - repo: https://github.com/stackabletech/airflow-operator
+        branch: "fix/dag-plugins" #"main"
```

Comment:
Alternatively wou could set up a separate  repository for this dag example.

Reply:
I'll open up a new issue for this.